src/backend/oracle-data-interface/Dockerfile

```diff
@@ -0,0 +1,31 @@
+#############################################################################################
+###              Stage where Docker is building spring boot app using maven               ###
+#############################################################################################
+FROM maven:3.8.3-jdk-11 as build
+
+ARG SKIP_TESTS=true
+ARG ENABLE_SPLUNK=false
+ARG SERVICE_NAME=oracle-data-interface-application
+
+WORKDIR /
+
+COPY . .
+
+RUN mvn -B clean package \
+		-Dtarget.fileName=${SERVICE_NAME} \
+        -Dmaven.test.skip=${SKIP_TESTS} \
+        -Denable-splunk=${ENABLE_SPLUNK}
+
+#############################################################################################
+
+#############################################################################################
+### Stage where Docker is running a java process to run a service built in previous stage ###
+#############################################################################################
+FROM eclipse-temurin:11
```

Comment:
Why is this the runtime image?  Other projects use openjdk11-jre-slim, ie https://github.com/bcgov/jag-pcss-criminal/blob/e48ed52cbed4cbc54da0788231bb53ecfb4db404/Dockerfile#L22

Reply:
I know the other projects use openjdk11-jre-slim but eclipse-temurin is the latest official build for the OpenJDK 11 binaries that I'm using to build the project. It's actually the same jdk (adopt open jdk) that other projects use except it is the latest version which has been moved to the Eclipse Foundation and rebranded as Adoptium Eclipse Temurin.